src/retry-queue.ts

```diff
@@ -4,6 +4,27 @@ import { CaptureMetrics } from './capture-metrics'
 import { QueuedRequestData, RetryQueueElement } from './types'
 import Config from './config'
 
+const thirtyMinutes = 30 * 60 * 1000
```

Comment:
So before we had a 60 second max, now we have 30x that as a max? Are we sure that makes sense?

Reply:
Before we had 3000*2^10 as the max which I think is 51 and a bit minutes?

I could easily be being slow